src/FileUpload/FileNameGenerator/Simple.php

```diff
@@ -56,11 +56,15 @@ public function getFileName($source_name, $type, $tmp_name, $index, $content_ran
      */
     protected function getUniqueFilename($name, $type, $index, $content_range)
     {
+        if (! is_array($content_range)) {
+            $content_range = [0];
+        }
```

Comment:
- To avoid the notice `Trying to access array offset on value of type int ` or `Trying to access array offset on value of type null` message on `PHP-7.4` version, it should do following approach:

1. Checking whether the `$content_range` variable is array.
If it's not array, assigning the `[0]` variable directly.